packages/ui/core-components/src/lib/molecules/dimension-grid/DimensionGrid.stories.svelte

```diff
@@ -100,3 +100,15 @@
 		})}
 	/>
 </Story>
+
+<Story name="string query with spaces">
+	<DimensionGrid
+		data={Query.create(
+			`SELECT "fare" as "fare price", "airline" as "air line" from flights limit 10`,
```

Comment:
Minor, but "air line" looks like a typo -- could we change to some other column name that also includes a space?  

```suggestion
			`SELECT "fare" as "fare price", "airline" as "Airline Name" from flights limit 10`,
```

Reply:
Nice, I like this better!